HelpSource/scdoc.js

```diff
@@ -105,7 +105,7 @@ function addInheritedMethods() {
             var m = mets[j];
             if(doc.methods.indexOf("_"+m.slice(1))<0) { // ignore methods already documented in this helpfile
                 var li = document.createElement("li");
-                li.innerHTML = "<a href='"+helpRoot+"/"+s.path+".html#"+m.slice(1)+"'>"+m.slice(2)+"</a>";
+                li.innerHTML = "<a href='"+helpRoot+"/"+s.path+".html#"+encodeURIComponent(m.slice(1))+"'>"+m.slice(2)+"</a>";
```

Comment:
@crucialfelix it seems  like ```escapeURIComponent``` is better for the portion of the url:

http://stackoverflow.com/questions/4540753/should-i-use-encodeuri-or-encodeuricomponent-for-encoding-urls

E.g. ```escapeURI``` doesn't encode "#", which is an issue @jamshark70 brought up elsewhere